indico/htdocs/sass/partials/_core.scss

```diff
@@ -104,3 +104,7 @@
     overflow-x: hidden;
     overflow-y: auto;
 }
+
+.truncate-text {
```

Comment:
As I mentioned already, `.truncate-text` doesn't convey to me that the text is going to be "ellipsisized". I would really go for `.ellipsisize`.